packages/react/runtime/src/lifecycle/ref/delay.ts

```diff
@@ -56,34 +64,41 @@ class RefProxy {
 
   constructor(refAttr: [snapshotInstanceId: number, expIndex: number]) {
     this.refAttr = refAttr;
+    this.task = undefined;
+
+    return new Proxy(this, {
```

Comment:
Can we add a codspeed benchmark to ref (cc @hzy ), I am not sure Proxy will slow down performance.